lib/components/dashboard/course-list/CourseList.tsx

```diff
@@ -227,105 +224,67 @@ const CourseList: FC<Props> = ({ mode }) => {
     const courseYearIndex: number = sourceYear.courses
       .map((c) => c._id)
       .indexOf(course._id);
-    try {
-      const resp = await axios.get(getAPI(window) + '/search', {
-        params: { query: course.number },
-      });
-      let retrievedCourses: SISRetrievedCourse[] = resp.data.data;
-      dispatch(updateCourseCache(retrievedCourses));
-      if (
-        retrievedCourses.length !== 0 &&
-        !checkDestValid(course, destination, retrievedCourses)
-      ) {
```

Comment:
this API call is not necessary because `/courses/dragged` route now does this check in backend